src/common/ceph_context.cc

```diff
@@ -425,7 +426,45 @@ void CephContext::do_command(std::string command, cmdmap_t& cmdmap,
         f->dump_string("option", *p);
       }
       f->close_section(); // unknown
-    } else if (command == "log flush") {
+    }
+    else if (command == "config diff get") {
+      std::string ceph_setting;
+      if (!cmd_getval(this, cmdmap, "var", ceph_setting)) {
+        f->dump_string("error", "syntax error: 'config diff get <var>'");
+      } //-- if (!cmd_getval(this, cmdmap, "var", ceph_setting))
```

Comment:
our code guidelines do not forbid this explicitly. and i agree this is a gray area, but see https://google.github.io/styleguide/cppguide.html#Implementation_Comments. the scope of the `if` block is obvious, imo.